### PR TITLE
added banner to readme 

### PR DESCRIPTION
**Pull request by cam-parra**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #+author: Cam Parra
 
-* Sovrin Ledger Payment plugins
+#+ATTR_HTML: :style margin-left: auto; margin-right: auto;
+[[./banner.png]]
+
+* Sovrin Token Plugins
```
Comment (burdettadam) [inline README.org]: did we decide on calling this "Sovrin Token Plugins"?

File: README.org [hunk 2]
```diff
@@ -1,6 +1,9 @@
 #+author: Cam Parra
 
-* Sovrin Ledger Payment plugins
+#+ATTR_HTML: :style margin-left: auto; margin-right: auto;
+[[./banner.png]]
```
Comment (burdettadam) [inline README.org]: what is banner.png?

File: README.org (hunk 1 above)
Comment (cam-parra) [reply README.org]: yes

File: README.org (hunk 2 above)
Comment (cam-parra) [reply README.org]: https://github.com/sovrin-foundation/sovrin/blob/master/banner.png

**Pull request merged**